acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -77,6 +78,18 @@ public void createKeyStoreYamlFileAt(
     createYamlFile(metadataFilePath, signingMetadata);
   }
 
+  public void createKeyStoreYamlFileAt(
```

Comment:
This function be reused by the createKeyStoreYamlFileAt(Path, BLSKeyPair, KdfFunction) variant as well to reduce some duplication.

Reply:
done I think